plugins/snapshots/erofs/loop.go

```diff
@@ -0,0 +1,53 @@
+//go:build linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package erofs
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/containerd/containerd/v2/core/mount"
+)
+
+func BindLoopDevice(file string) (string, error) {
```

Comment:
This doesn't need to be exported.

Reply:
Hi @ktock , thanks. very sorry about my limited Go skills... I will convert the function name started with an uppercase

Reply:
done